src/components/Settings/ApplicationSettingsScreen.js

```diff
@@ -235,7 +235,7 @@ const ApplicationSettingsScreen = ({
 
       <SettingsBuildItem
         label={intl.formatMessage(messages.network)}
-        value={CONFIG.NETWORK}
+        value={'Byron Mainnet / Shelley Testnet'}
```

Comment:
How temporary are these? If we think this will stick around for a while, maybe it's better to make this a translatable string? Or maybe we already have strings for "byron" and "shelley we can re-use to construct this string?

Reply:
We need some refactoring to support both Shelley testnet and mainnet. It's probably a good idea to do it before working on new features (like tx history, etc). So maybe next week?

I think it's safe to add that only to patch the coming release.